components/tripStops/TripStopLocationWizard.tsx

```
@@ -0,0 +1,153 @@
+            keyboardBehavior="interactive"
+            keyboardBlurBehavior="restore"
+            android_keyboardInputMode="adjustResize"
+            snapPoints={["50%", "80%"]}
+            onClose={onClose}
+        >
+            <BottomSheetView style={{ flex: 1 }}>
+                <View>
+                    <View className="flex-row">
+                        <IconSymbol name="map" color={colors.text} />
+                        <Text className="text-lg font-bold dark:text-white">
+                            Adresse de l'étape {tripStop?.name}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)